### PR TITLE
Changed the endpoint variable to be dependent on GKE cluster state

### PR DESCRIPTION
**Pull request by kevensen**
Addresses #212

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,78 +122,6 @@ Version 1.0.0 of this module introduces a breaking change: adding the `disable-l
 In either case, upgrading to module version `v1.0.0` will trigger a recreation of all node pools in the cluster.
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-## Inputs
```
Comment (morgante) [inline README.md]: Why is this content being removed?

Comment (kevensen) [reply README.md]: I ran `make generate`.  I forgot I need to do `make generate_docs`.

**Comment by kevensen** [comment]
@morgante My commit was removing the inputs due to me not running `make generate_docs`.  That's been corrected.

Not sure why the lint tests are failing.

**Comment by morgante** [comment]
@kevensen Can you unstage all README changes? Something is funky with your `make generate_docs` and your PR shouldn't need to update the README anyways.

**Comment by kevensen** [comment]
When I run `make generate` it updates the README.md's.

I've backed out the changes to the README.md's.

**Comment by kevensen** [comment]
@morgante Thank you!

**Pull request merged**